Taskfile.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+includes:
+  agents:
+    taskfile: ./agents/Taskfile.yml
+    dir: ./agents
+
+  server:
+    taskfile: ./server/Taskfile.yml
+    dir: ./server
+
+tasks:
+  build:
+    desc: build all
```

Comment:
is this only useful in cicd?

Reply:
It's equal to `make build`.

Reply:
but if one extension exists in multiple folders, will it be installed for multiple times?